service/src/main/java/org/cbioportal/genome_nexus/service/mixin/IntergenicConsequencesMixin.java

```diff
@@ -0,0 +1,19 @@
+package org.cbioportal.genome_nexus.service.mixin;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class IntergenicConsequencesMixin
+{
+    @JsonProperty(value="impact", required = true)
+    private String impact;
+
+    @JsonProperty(value="variantAllele", required = true)
```

Comment:
good to keep the original vep field for the user. that's what we do with the other vep mixin classes as well (i.e. in this case `variant_allele`)